templates/cli/src/Client.php.twig

```diff
@@ -0,0 +1,324 @@
+<?php
+
+namespace {{ spec.title | caseUcfirst }};
+
+use Exception;
+use Utopia\CLI\Console;
+
+class Client
+{
```

Comment:
We need to make sure we support global headers just as @lohanidamodar did here: https://github.com/appwrite/sdk-generator/pull/96

Reply:
Done